### ISSUE TITLE
[jacobsen on 2010-02-02 17:30:42] : Logging setup failure, which kills runs

### ISSUE THREAD

**Issue by dglo** [dglo/pdaq#1274]
No associated GitHub commit

**Comment by dglo** [comment]
[jacobsen on 2010-02-02 17:36:42]
What we found today (Sabrina and I) is that changing the run number to a run which had not been run before (based on the directories in /mnt/data/pdaq/log) seemed to avoid the problem (at least the first and only time we've tried so far).

I suspect there is something slightly goofy with the 'move-existing-log-directories-away' logic in DAQRun.py.  That stuff has been refactored since I wrote the original version.

Since we changed the run number once with ExpControlSkel, it should 'remember' the new sequence and avoid that problem for future runs... if, in fact, it actually WAS the problem.

Here is the command:

./ExpControlSkel.py -c sps-partition-parallel-strings-8-15-23-v2 -n1 -d 1200 -s198\n\n
